browser/resources/settings/brave_sync_page/brave_sync_page.ts

```diff
@@ -81,6 +85,16 @@ export class SettingsBraveSyncPageElement extends SettingsBraveSyncPageElementBa
     return this.i18n(key);
   }
 
+  computeShouldShowCustomSyncInput_() {
+    const isAlreadySetup =
+      this.syncStatus_ !== undefined && !this.syncStatus_.firstSetupInProgress
+
+    return !(
+      isAlreadySetup &&
+      this.braveBrowserProxy_.getCustomSyncUrlAtStartup() === ''
```

Comment:
So `this.braveBrowserProxy_.getCustomSyncUrlAtStartup() === ''` is only true if the custom sync URL is empty.

My question would be, does this checking the custom sync URL also show values set by a command line switch or by policy?

Reply:
No, the block will be visible when the custom sync URL is set either via the command line or by policy.

Reply:
> The block will be visible when the custom sync URL is set by policy(non empty).
> 
> The value passed from the command line will not be shown in this block, so it won't have any effect here.

Alright, thanks for the info! 🙂